server/utils.js

```diff
@@ -39,47 +40,58 @@ module.exports = {
 
     // by default it sorts with 'relevance' and returns only 10 items (numItems=10)
     request.get(url, function(err, response, body) {
+      if (err) callback(err, null)
       if (!err && response.statusCode === 200) {
         var products = JSON.parse(body)
         // products.items is an array of items
-        callback(products.items)
+        callback(null, products.items)
       }
     })
   },
 
 
   sendEmail: (userEmail) => {
-    var transporter = nodemailer.createTransport({
-     service: 'gmail',
-     auth: {
-        user: config.EMAIL_USER, //repulsiv.hr@gmail.com
-        pass: config.EMAIL_PASS
+
+    let transporter = nodemailer.createTransport({
+      host: 'smtp.gmail.com',
+      port: 465,
+      secure: true,
+      auth: {
+          type: 'OAuth2',
+          clientId: config.GMAIL_CLIENT_ID,
+          clientSecret: config.GMAIL_SECRET
       }
-    });
+  });
 
-    const mailOptions = {
-      from: config.EMAIL_USER,
+    mailOptions = {
```

Comment:
do you want to make it local, looks like mailOptions is global

Reply:
yes, good catch. Let me fix